packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -209,6 +225,19 @@ export class Cluster extends Resource implements ICluster {
       this.enableFargateCapacityProviders();
     }
 
+    if (props.defaultCapacityProviderStrategy) {
+      props.defaultCapacityProviderStrategy.forEach(dcp => {
+        if (!this._capacityProviderNames.includes(dcp.capacityProvider)) {
```

Comment:
`capacityProviders` was deprecated in v1 and should not be included in the logic of v2.

Reply:
I have one question regarding the validations.
If it's better to provide validations on default capacity providers or to allow customers to create potentially failing stacks by passing strings as a defaultCapacityProvider?